be/src/simd/simd.h

```diff
@@ -53,6 +53,44 @@ inline size_t count_zero(const int8_t* data, size_t size) {
     return count;
 }
 
+inline size_t count_zero_int32(const uint32_t* data, size_t size) {
+    size_t count = 0;
+    const uint32_t* end = data + size;
+
+#if defined(__SSE2__) && defined(__POPCNT__)
+    // count per 16 int
+    const __m128i zero16 = _mm_setzero_si128();
+    const uint32_t* end16 = data + (size / 16 * 16);
+
+    for (; data < end16; data += 16) {
+        count += __builtin_popcountll(static_cast<uint64_t>(_mm_movemask_ps(_mm_cvtepi32_ps(_mm_cmpeq_epi32(
+                                              _mm_loadu_si128(reinterpret_cast<const __m128i*>(data)), zero16)))) |
+                                      (static_cast<uint64_t>(_mm_movemask_ps(_mm_cvtepi32_ps(_mm_cmpeq_epi32(
+                                               _mm_loadu_si128(reinterpret_cast<const __m128i*>(data + 4)), zero16))))
+                                       << 4u) |
```

Comment:
`<< 16u`? The result of `(_mm_movemask_ps(_mm_cvtepi32_ps(_mm_cmpeq_epi32( 
                                               _mm_loadu_si128(reinterpret_cast<const __m128i*>(data + 4)), zero16))))` is 16-bit.

Reply:
each `_mm_movemask_ps` get 4 bits results corresponding to 4 int32, so here is 4, 8, 12